### PR TITLE
testing/hylang: new aport

### PR DESCRIPTION
**Pull request by nmeum**
Includes a bunch of previously unpackaged dependencies. Many of those still needs some work, therefore this is just WIP for now.

### CODE REVIEW

File: testing/py3-astor/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
+# Maintainer:
+pkgname=py3-astor
+pkgver=0.8
+pkgrel=0
+pkgdesc="Easy manipulation of Python source via the AST"
+url="https://github.com/berkerpeksag/astor"
+arch="all"
```
Comment (maxice8) [inline testing/py3-astor/APKBUILD]: noarch

File: testing/py3-astor/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+license="BSD-3-Clause-Clear"
+depends="python3"
+makedepends="py3-setuptools"
+checkdepends="py3-nose"
+options="!check" # TODO
```
Comment (maxice8) [inline testing/py3-astor/APKBUILD]: Tests run fine here.

```
.............S.........S............S.................
----------------------------------------------------------------------
Ran 54 tests in 31.429s

OK (SKIP=3)
```

File: testing/py3-clint/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
```
Comment (maxice8) [inline testing/py3-clint/APKBUILD]: Patch for py3-clint: http://ix.io/1TXS

- Enables tests with py3-pytest (command extracted from their tox.ini)
- Sets noarch

File: testing/py3-rply/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+subpackages=""
+source="$pkgname-$pkgver.tar.gz::https://github.com/alex/rply/archive/v$pkgver.tar.gz"
+builddir="$srcdir/rply-$pkgver"
+
+build() {
+	python3 setup.py build
+}
+
+check() {
+	python3 setup.py test
```
Comment (maxice8) [inline testing/py3-rply/APKBUILD]: This is a trap.

```
running build_ext

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
```

Their tox.ini uses py.test

File: testing/py3-rply/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
```
Comment (maxice8) [inline testing/py3-rply/APKBUILD]: Proposed patch: http://ix.io/1TXT

- Uses py3-pytest to run tests
- Set noarch

**Comment by nmeum** [comment]
Actually started using this programming language. Might therefore as well maintain it for Alpine. @maxice8 thanks a lot for your feedback, addressed it and applied your patches. Did you notice anything else?

**Comment by maxice8** [comment]
LGTM

**Pull request closed without merging**